[Improvement] Fix Javadoc issues in task :authorizations:authorization-ranger:javadoc

### What would you like to be improved?

> Task :authorizations:authorization-ranger:javadoc
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:156: warning: no @param for metadataObject
  protected abstract RangerPolicy createPolicyAddResources(
                                  ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:160: warning: no @param for authzMetadataObject
  protected abstract List<RangerPolicy> wildcardSearchPolies(
                                        ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:160: warning: no @return
  protected abstract List<RangerPolicy> wildcardSearchPolies(
                                        ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:183: warning: no @param for authzMetadataObject
  protected RangerPolicy preciseFindPolicy(
                         ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:183: warning: no @param for preciseFilters
  protected RangerPolicy preciseFindPolicy(
                         ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:183: warning: no @return
  protected RangerPolicy preciseFindPolicy(
                         ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:992: warning: no @param for authzMetadataObject
  protected abstract void renameMetadataObject(
                          ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:992: warning: no @param for newAuthzMetadataObject
  protected abstract void renameMetadataObject(
                          ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:1030: warning: no @param for object
  public abstract AuthorizationSecurableObject generateAuthorizationSecurableObject(
                                               ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:1030: warning: no @param for privileges
  public abstract AuthorizationSecurableObject generateAuthorizationSecurableObject(
                                               ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java:1030: warning: no @return
  public abstract AuthorizationSecurableObject generateAuthorizationSecurableObject(
                                               ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHadoopSQLSecurableObject.java:41: warning: no @param for privileges
  public RangerHadoopSQLSecurableObject(
         ^
/Users/justin/gravitino/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHelper.java:257: warning: no @return
  protected RangerRole createRangerRoleIfNotExists(String roleName, boolean isOwnerRole) {
         

### How should we improve?

Fix Javadoc issues

--- a/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java
+++ b/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java
@@ -151,12 +151,18 @@ public abstract class RangerAuthorizationPlugin
   /**
    * Create a new policy for metadata object
    *
+   * @param metadataObject The authorization metadata object to create the policy for.
    * @return The RangerPolicy for metadata object.
    */
   protected abstract RangerPolicy createPolicyAddResources(
       AuthorizationMetadataObject metadataObject);
 
-  /** Wildcard search the Ranger policies in the different Ranger service. */
+  /**
+   * Wildcard search the Ranger policies in the different Ranger service.
+   *
+   * @param authzMetadataObject The authorization metadata object used to perform wildcard search.
+   * @return A list of Ranger policies matching the wildcard conditions.
+   */
   protected abstract List<RangerPolicy> wildcardSearchPolies(
       AuthorizationMetadataObject authzMetadataObject);
 
@@ -179,6 +185,13 @@ public abstract class RangerAuthorizationPlugin
    * wildcard(*,?) conditions, If you use `db.table` condition to search policy, the Ranger will
    * match `db1.table1`, `db1.table2`, `db*.table*`, So we need to manually precisely filter this
    * research results.
+   *
+   * @param authzMetadataObject The authorization metadata object used as the base for searching
+   *     policies.
+   * @param preciseFilters A map of resource keys and their expected exact values for filtering the
+   *     policies.
+   * @return The matched Ranger policy if found, or {@code null} if none matches.
+   * @throws AuthorizationPluginException If multiple policies are found or a validation fails.
    */
   protected RangerPolicy preciseFindPolicy(
       AuthorizationMetadataObject authzMetadataObject, Map<String, String> preciseFilters)
@@ -988,6 +1001,9 @@ public abstract class RangerAuthorizationPlugin
    * IF rename the TABLE, Need to rename these the relevant policies, `{schema}.*`, `{schema}.*.*`
    * <br>
    * IF rename the COLUMN, Only need to rename `{schema}.*.*` <br>
+   *
+   * @param authzMetadataObject The original metadata object to be renamed.
+   * @param newAuthzMetadataObject The new metadata object after renaming.
    */
   protected abstract void renameMetadataObject(
       AuthorizationMetadataObject authzMetadataObject,
@@ -1026,7 +1042,13 @@ public abstract class RangerAuthorizationPlugin
   @Override
   public void close() throws IOException {}
 
-  /** Generate authorization securable object */
+  /**
+   * Generate authorization securable object.
+   *
+   * @param object The authorization metadata object to base the securable object on.
+   * @param privileges The set of privileges associated with the securable object.
+   * @return The generated authorization securable object.
+   */
   public abstract AuthorizationSecurableObject generateAuthorizationSecurableObject(
       AuthorizationMetadataObject object, Set<AuthorizationPrivilege> privileges);
 

--- a/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHadoopSQLSecurableObject.java
+++ b/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHadoopSQLSecurableObject.java
@@ -37,6 +37,7 @@ public class RangerHadoopSQLSecurableObject extends RangerHadoopSQLMetadataObjec
    * @param parent The parent of the metadata object
    * @param name The name of the metadata object
    * @param type The type of the metadata object
+   * @param privileges The set of privileges assigned to this securable object
    */
   public RangerHadoopSQLSecurableObject(
       String parent,

--- a/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHelper.java
+++ b/authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHelper.java
@@ -253,6 +253,7 @@ public class RangerHelper {
    *
    * @param roleName The role name to create
    * @param isOwnerRole The role is owner role or not
+   * @return The created or existing RangerRole
    */
   protected RangerRole createRangerRoleIfNotExists(String roleName, boolean isOwnerRole) {
     roleName = generateGravitinoRoleName(roleName);

